app/model/editions/EditionsArticle.scala

```diff
@@ -1,35 +1,72 @@
 package model.editions
 
-import com.gu.editions.{PublishedArticle, PublishedFurniture}
+import com.gu.editions.{PublishedArticle, PublishedFurniture, MediaUrl}
 import play.api.libs.json.Json
 import scalikejdbc.WrappedResultSet
 
-case class ArticleMetadata(
+case class Image (
```

Comment:
This and `MediaUrl` would be the same model - need to confirm with @AWare about the structure needed.